src/main/java/com/programmers/voucher/controller/VoucherController.java

```diff
@@ -0,0 +1,29 @@
+package com.programmers.voucher.controller;
+
+import com.programmers.voucher.domain.Voucher;
+import com.programmers.voucher.domain.VoucherMapper;
+import com.programmers.voucher.dto.VoucherRequestDto;
+import com.programmers.voucher.dto.VoucherResponseDto;
+import com.programmers.voucher.service.VoucherService;
+import org.springframework.stereotype.Controller;
+
+import java.util.List;
+
+@Controller
+public class VoucherController {
+
+    private final VoucherService voucherService;
+
+    public VoucherController(VoucherService voucherService) {
+        this.voucherService = voucherService;
+    }
+
+    public VoucherResponseDto create (VoucherRequestDto requestDto) {
+        Voucher voucher = voucherService.create(requestDto);
+        return VoucherMapper.convertDomainToResponseDto(voucher);
+    }
+
+    public List<VoucherResponseDto> findAll() {
+        return voucherService.findVouchers().stream().map(VoucherMapper::convertDomainToResponseDto).toList();
```

Comment:
```suggestion
        return voucherService.findVouchers().stream()
		        .map(VoucherMapper::convertDomainToResponseDto)
		        .toList();
```
별건 아니지만 스트림을 사용할 때는 이렇게 작성하는 편이 가독성이 좋은 것 같아요